src/test/java/org/sagebionetworks/bridge/workers/dynamodb/streams/StreamsUtilsTest.java

```diff
@@ -0,0 +1,5 @@
+package org.sagebionetworks.bridge.workers.dynamodb.streams;
+
+public class StreamsUtilsTest {
```

Comment:
This shouldn't be checked in.  Mocking around is expensive.  That's why it's empty.  But StreamsUtils has been manually tested.